src/components/NftRewards/NftTierCard.tsx

```diff
@@ -55,11 +54,13 @@ export function NftTierCard({
     setPreviewVisible(true)
   }
 
-  const remainingSupplyText =
-    rewardTier?.remainingSupply !== undefined &&
-    rewardTier.remainingSupply < MAX_REMAINING_SUPPLY
-      ? t`${rewardTier?.remainingSupply} remaining`
-      : t`Unlimited`
+  const remainingSupply = rewardTier?.remainingSupply
+  const hasRemainingSupply = remainingSupply && remainingSupply > 0
+  const remainingSupplyText = !hasRemainingSupply
+    ? t`SOLD OUT`
+    : rewardTier.maxSupply === DEFAULT_NFT_MAX_SUPPLY
```

Comment:
@johnnyd-eth note also I changed this to `maxSupply`, and used `DEFAULT_NFT_MAX_SUPPLY` here (which is 999,999,999) - seems to work for me.